### PR TITLE
onboarding for designers tweaks

### PR DESCRIPTION
**Pull request by rrhyne**
## Design section changes
* Defines design people and roles
* Updates onboarding for product designers
* Creates a new product design manager onboarding section of the design onboarding page

## Product onboarding changes
* Alters manager onboarding steps to link to handbook instead of copy handbook to onboarding doc
* Moves setting up the design environment from the product onboarding page to the product design onboarding page

### CODE REVIEW

File: content/departments/product-engineering/product/design/design_process.md [hunk 1]
```diff
@@ -123,14 +122,14 @@ While asynchronous communication is a core attribute of remote work, key moments
...
-  - Signed in vs. anonymous user
+  - Server / Cloud
+  - If Cloud: Signed in vs. anonymous users
   - User permissions
-  - Interactive mode / plain text mode
 - Process
   - A GitHub issue should be created to track the work
   - Designs are produced in Figma
-  - Designs will be announced in Slack and linked in the GitHub issue well before they are complete for review
+  - Let others know about your designs in your team Slack channel
```
Comment (rrhyne) [inline content/departments/product-engineering/product/design/design_process.md]:
```suggestion
  - Let others know about your designs in your team Slack channel and in the product-design channel and other places of interest
```

File: content/departments/product-engineering/product/design/design_process.md [hunk 2]
```diff
@@ -123,14 +122,14 @@ While asynchronous communication is a core attribute of remote work, key moments
...
+  - Linked to your designs in the GitHub issue before they are complete to promote designing in the open
```
Comment (rrhyne) [inline content/departments/product-engineering/product/design/design_process.md]:
```suggestion
  - Link to your designs in the GitHub issue before they are complete to promote designing in the open
```

File: content/departments/product-engineering/product/onboarding/index.md [hunk 3]
```diff
@@ -36,13 +37,15 @@ Remember:
   - One of our values is to be [open and transparent](../../../../company-info-and-process/values/index.md#open-and-transparent)
   - Good reasons for things to be private: sensitive customer information, sensitive personal information
   - If you see a "request permission" page for some doc or other resource, it is a mistake. Just ask to be granted access to it (and make a PR to update the handbook so the next person starting is granted access to that system).
+  - When you have a question, try to ask publically. Other people probably have the same question.
```
Comment (rrhyne) [inline content/departments/product-engineering/product/onboarding/index.md]:
```suggestion
  - When you have a question, try to ask publicly. Other people probably have the same question.
```

**Review by marisak — APPROVED** [review]
Looks great! Thanks for making all these edits and consolidating so much information!

**Pull request merged**